model/Export/Extractor/OntologyExtractorRunner.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ *
+ */
+
+namespace oat\taoQtiItem\model\Export\Extractor;
+
+use core_kernel_classes_Property;
+use core_kernel_classes_Resource;
+use oat\taoQtiItem\model\flyExporter\extractor\OntologyExtractor;
+
+class OntologyExtractorRunner
+{
+    /**
+     * @param core_kernel_classes_Resource $item
+     * @param core_kernel_classes_Property $classProperty
+     *
+     * @return HashEntry
+     */
+    public static function run(core_kernel_classes_Resource $item, core_kernel_classes_Property $classProperty)
+    {
+        $itemProperty = $item->getProperty($classProperty->getUri());
+        $labelItemProperty = $itemProperty->getLabel();
+
+        $extractor = new OntologyExtractor();
+        $extractor->addColumn($labelItemProperty, ['property' => $classProperty->getUri()]);
```

Comment:
could you please modify the OntologyExtractor so it takes the uri instead of the label.
example : uriProperty1=>labelA, uriProperty2=>labelA, uriProperty3=>labelA you should see 3 columns with the same labelA, for now you will have only 1 column with the last call

Reply:
Did not supported, so I modified to add a suffix (1)..(n) if the column with the same name exists